src/engine/effects/engineeffectchain.cpp

```diff
@@ -165,98 +165,77 @@ void EngineEffectChain::process(const ChannelHandle& handle,
 
     EffectProcessor::EnableState effectiveEnableState = channel_info.enable_state;
 
+    if (channel_info.enable_state == EffectProcessor::DISABLING) {
+        channel_info.enable_state = EffectProcessor::DISABLED;
```

Comment:
Mmm.. What is done in music composing sometimes is that the effects run all the time, and the input volume to the effect is controlled.